Mathlib/Algebra/CovariantAndContravariant.lean

```diff
@@ -328,62 +347,62 @@ theorem contravariant_flip_iff [IsSymmOp N N mu] :
   elim := (covariant_flip_iff N r (· * ·)).mpr CovariantClass.elim
 
 @[to_additive]
 instance contravariant_swap_mul_of_contravariant_mul [CommSemigroup N]
-    [ContravariantClass N N (· * ·) r] : ContravariantClass N N (swap (· * ·)) r where
+    [ContravariantClass N N (*) r] : ContravariantClass N N (swap (*)) r where
   elim := (contravariant_flip_iff N r (· * ·)).mpr ContravariantClass.elim
 
-theorem covariant_lt_of_covariant_le_of_contravariant_eq [ContravariantClass M N μ (· = ·)]
-    [PartialOrder N] [CovariantClass M N μ (· ≤ ·)] : CovariantClass M N μ (· < ·) where
+theorem covariant_lt_of_covariant_le_of_contravariant_eq [ContravariantClass M N μ Eq]
+    [PartialOrder N] [CovariantClass M N μ LE.le] : CovariantClass M N μ LT.lt where
   elim a _ _ bc := (CovariantClass.elim a bc.le).lt_of_ne (bc.ne ∘ ContravariantClass.elim _)
 
 theorem contravariant_le_of_contravariant_eq_and_lt [PartialOrder N]
-    [ContravariantClass M N μ (· = ·)] [ContravariantClass M N μ (· < ·)] :
-    ContravariantClass M N μ (· ≤ ·) where
+    [ContravariantClass M N μ Eq] [ContravariantClass M N μ LT.lt] :
+    ContravariantClass M N μ LE.le where
   elim := (contravariant_le_iff_contravariant_lt_and_eq M N μ).mpr
     ⟨ContravariantClass.elim, ContravariantClass.elim⟩
 
 /- TODO:
-  redefine `IsLeftCancel N mu` as abbrev of `ContravariantClass N N mu (· = ·)`,
-  redefine `IsRightCancel N mu` as abbrev of `ContravariantClass N N (flip mu) (· = ·)`,
+  redefine `IsLeftCancel N mu` as abbrev of `ContravariantClass N N mu Eq`,
+  redefine `IsRightCancel N mu` as abbrev of `ContravariantClass N N (flip mu) Eq`,
   redefine `IsLeftCancelMul` as abbrev of `IsLeftCancel`,
   then the following four instances (actually eight) can be removed in favor of the above two. -/
 
 @[to_additive]
 instance LeftCancelSemigroup.covariant_mul_lt_of_covariant_mul_le [LeftCancelSemigroup N]
-    [PartialOrder N] [CovariantClass N N (· * ·) (· ≤ ·)] :
-    CovariantClass N N (· * ·) (· < ·) where
+    [PartialOrder N] [CovariantClass N N (· * ·) LE.le] :
+    CovariantClass N N (· * ·) LT.lt where
   elim a _ _ bc := (CovariantClass.elim a bc.le).lt_of_ne ((mul_ne_mul_right a).mpr bc.ne)
 
 @[to_additive]
 instance RightCancelSemigroup.covariant_swap_mul_lt_of_covariant_swap_mul_le
-    [RightCancelSemigroup N] [PartialOrder N] [CovariantClass N N (swap (· * ·)) (· ≤ ·)] :
-    CovariantClass N N (swap (· * ·)) (· < ·) where
+    [RightCancelSemigroup N] [PartialOrder N] [CovariantClass N N (swap (· * ·)) LE.le] :
+    CovariantClass N N (swap (· * ·)) LT.lt where
   elim a _ _ bc := (CovariantClass.elim a bc.le).lt_of_ne ((mul_ne_mul_left a).mpr bc.ne)
 
 @[to_additive]
 instance LeftCancelSemigroup.contravariant_mul_le_of_contravariant_mul_lt [LeftCancelSemigroup N]
-    [PartialOrder N] [ContravariantClass N N (· * ·) (· < ·)] :
-    ContravariantClass N N (· * ·) (· ≤ ·) where
+    [PartialOrder N] [ContravariantClass N N (· * ·) LT.lt] :
```

Comment:
Is there a reason why this one is not `(*)`?

Reply:
I think that this is an oversight.  It might be caused by the fact that the regex that I used assumed that it should match `(. * .)` and `(. < .)`, rather than one expanded and one not.

This was a reasonable assumption before doing the replacement, but might have gotten confused after some partial fix of a merge conflict.

I'll fix this and the regex once I'm back at a computer.

However, the bigger issue here is that introducing the `(*)` syntax is very nice, but gets used *everywhere* in place of the infix notation.

Reply:
I replaced the one you highlighted and a couple more that had creeped in due to the hybrid usage `(· * ·) LT.lt`/`(· * ·) LE.le` and analogous with `+`.  I could not find cases with the "dual" `HMul.hMul (· < ·)` and variations with `Add/≤`.